### PR TITLE
CSS highlight improvement

### PR DESCRIPTION
**Pull request by BKR-dev**
@vrischmann so i was thinking i quickly add something to make the css stuff look nicer as mentioned in issue https://github.com/vrischmann/tree-sitter-templ/issues/9

i think i got it improved a little bit, although im not sure how to make it better for dynamic_class_attribute_value to reflect the parts of that statement, i would like that each call_expression looks like component_identifier to give it a coherent look.

### CODE REVIEW

**Comment by BKR-dev** [comment]
Just wondering if you had the chance to take a look at it @vrischmann ? Thanks in advance 🤗

**Comment by vrischmann** [comment]
Hi, not yet. I'll try to take a look this weekend.

File: queries/templ/highlights.scm [hunk 1]
```diff
@@ -46,8 +46,11 @@
 (element_text) @string.special
 (style_element_text) @string.special
 
+(css_identifier) @keyword
```
Comment (vrischmann) [inline queries/templ/highlights.scm:49]: This change makes the CSS component declaration look like a keyword, which clashes with a component declaration:
image
I think it's better to use `@function` here.

File: queries/templ/highlights.scm [hunk 2]
```diff
@@ -46,8 +46,11 @@
...
 (css_property
   name: (css_property_name) @attribute)
+(css_property
+  name: (css_property_value) @string)
```
Comment (vrischmann) [inline queries/templ/highlights.scm:53]: This breaks highlighting because you're duplicating a rule for `name`, you need `value`.

**Review by vrischmann — CHANGES_REQUESTED** [review]

**Comment by vrischmann** [comment]
A couple of changes needed. It's not perfect but I'm fine highlighting the property value as a string.

In the past what I wanted to do for highlighting the CSS properties was to directly inject the CSS highlightings queries, just like it's done for the `<style>` element [here](https://github.com/vrischmann/tree-sitter-templ/blob/master/queries/templ/injections.scm#L4), however I haven't been able to make it work yet.

**Comment by BKR-dev** [comment]
Thanks for the feedback, i made the changes you mentioned. My main goal is to give it some highlighting to make it nicer to read, i wasnt sure what to do exactly hence my "just go for it" pr. 

i looked at what you linked but im afraid i dont understand tbh. i try to grasp the things one at a time as im working on it and thinking "hey, that would be nice if xzy" 

Thank you for taking your time and helping me out, appreciate it!

File: queries/templ/highlights.scm (hunk 1 above)
Comment (BKR-dev) [reply queries/templ/highlights.scm]: changed to function

File: queries/templ/highlights.scm (hunk 2 above)
Comment (BKR-dev) [reply queries/templ/highlights.scm]: fixed

**Comment by vrischmann** [comment]
Sure, don't worry about it :) I'll merge this PR as is.

Injection will wait but if you're curious about it, the [official documentation](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection) is good.

**Review by vrischmann — APPROVED** [review]

**Pull request merged**